hyperspy/tests/signal/test_tools.py

```diff
@@ -593,14 +593,14 @@ def test_valuemax(self):
 
     def test_rebin(self):
         s = self.s
-        new_shape = (3, 2, 1, 3)
+        scale = (1, 2, 1, 2)
         if self.s._lazy:
```

Comment:
Don't know I was just leaving requirements like that to match the original code.